packages/maskbook/src/plugins/Trader/apis/trending/prediction_market_tokens.json

```diff
@@ -0,0 +1,36 @@
+[
```

Comment:
Hi, move this file to `apis/uniswap/` folder.

Reply:
okay